packages/angular/src/schematics/stories/stories.ts

```diff
@@ -215,6 +231,12 @@ export function createAllStories(
           }
         });
 
+        if (generateCypressSpecs && !e2eProject) {
+          context.logger.info(
+            'There was no e2e project configured, so cypress specs will not be generated.'
```

Comment:
Can we add the `e2eProject` to the console output? Like 

```
context.logger.info(
  `There was no e2e project "${e2eProjectName}" found, so cypress specs will not be generated. Pass "--cypressProject" to specify a different e2e project name`
)
```

Or something similar. This should help people understand what went wrong if they forgot the flag or something.

Reply:
Oh sure!!